zfmt/ast.go

```diff
@@ -64,7 +64,7 @@ func (c *canon) defs(defs []ast.Def, separator string) {
 }
 
 func (c *canon) def(d ast.Def) {
-	c.write(d.Name)
+	c.expr(d.Name, "")
```

Comment:
Nit: `write` is clearer. (Same for all the other places below where `write` is changed to `expr`.)

```suggestion
	c.write(d.Name.Name, "")
```

Reply:
You are probably right but I do find `d.Name.Name` to be really jarring to look at.

Reply:
Agreed. We could address that (later) by renaming `ast.ID.Name` to `Text`.